test/fixtures/scale.linear/tiny-numbers.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
One thing I noticed is that we lose the test description when it's a fixture. Maybe we should add it here as a comment?

> // Should generate max and min that are not equal when data contains values that are very close to each other

Reply:
done